doc/configuration.md

```diff
@@ -57,12 +65,36 @@ _Note: all fields are optional unless otherwise marked_
     * **mode** (integer): the file's permission mode.
     * **user** (object): specifies the file's owner.
       * **id** (integer): the user ID of the owner.
+      * **name** (string): the user name of the owner.
     * **group** (object): specifies the group of the owner.
       * **id** (integer): the group ID of the owner.
+      * **name** (string): the group name of the owner.
+  * **directories** (list of objects): the list of directories to be created.
+    * **filesystem** (string, required): the internal identifier of the filesystem in which to create the directory. This matches the last filesystem with the given identifier.
+    * **path** (string, required): the absolute path to the directory.
+    * **mode** (integer): the directory's permission mode. Note that the mode must be properly specified as a **decimal** value (i.e. 0755 -> 493).
```

Comment:
Not necessary with yaml; you can specify numbers in octal. Drop the part about decimal value.